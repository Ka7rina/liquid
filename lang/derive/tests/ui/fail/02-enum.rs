use liquid_lang as liquid;

use liquid::InOut;
#[derive(InOut)]
enum MyStruct {
    U32(u32),
    S(String),
}

fn main() {}
