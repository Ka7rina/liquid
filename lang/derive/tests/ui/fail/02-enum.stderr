error: enums are not supported
 --> $DIR/02-enum.rs:5:1
  |
5 | / enum MyStruct {
6 | |     U32(u32),
7 | |     S(String),
8 | | }
  | |_^
